pygccxml/parser/source_reader.py

```diff
@@ -242,9 +243,12 @@ def create_xml_file(self, source_file, destination=None):
             for line in process.stdout.readlines():
                 if line.strip():
                     results.append(line.rstrip())
+            for line in process.stderr.readlines():
+                if line.strip():
+                    results.append(line.rstrip())
 
             exit_status = process.returncode
-            msg = os.linesep.join([str(s) for s in results])
+            msg = os.linesep.join([str(s.decode()) for s in results])
```

Comment:
Error message is returned as string but process pipes return as bytes. Decoding is necessary for reasonable looking output